build-config/make/u-boot.make

```diff
@@ -10,7 +10,7 @@
 # makefile fragment that defines the build of the onie cross-compiled U-Boot
 #
 
-UBOOT_VERSION		?= 2013.01.01
+UBOOT_VERSION		?= 2015.10-rc1
```

Comment:
This is a platform specific change being made to core file shared by many boards.  Instead of setting UBOOT_VERSION here, set it in the machine.make of your platform.